.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,9 +6,7 @@ labels: ''
 assignees: ''
 ---
 
-Please fill in the following information to report a problem with Cantera.
-If you have a question about using Cantera, please post it on our
-[Google Users' Group](https://groups.google.com/forum/#!forum/cantera-users).
+<!-- Please fill in the following information to report a problem with Cantera. If you have a question about using Cantera, please post it on our [Google Users' Group](https://groups.google.com/forum/#!forum/cantera-users). -->
```

Comment:
Since the text in the HTML comments will not show up as a clickable link, I'd suggest removing the corresponding Markdown formatting.

This would also be a good place to add a link to the Enhancements repository.

Reply:
I also enclosed additional instructions in HTML comments.